added the Get Container operation

hitohata opened the pull request:
## Overviews of implementation

Added a get container operation

## Review points

Please check the tests

hitohata committed:
feat: added an add container test

hitohata committed:
feat: added a get container operation

hitohata committed:
chore: format

hitohata committed:
chore: removed unused modules

taniguchiiqqq commented on backend/main/src/Shared/Adapters/DB/NishikiDBTypes.ts:
```diff
@@ -43,3 +43,58 @@ export type InvitationLink = {
+ */
+export type FoodItem = {
+	FoodId: string;
+	Name: string | null;
```
I think the type of  `Name` should be string because the name is a required prop. Or is there any case it can be null if we use dynamoDB?

hitohata replied:
Thank you. The reason why I wrote it is it so is written in the document.
https://genesis-tech-tribe.github.io/nishiki-documents/project-document/database#foods

You are right, the name of the food is mandatory. I'll fix it and the document also.

taniguchiiqqq replied:
OMG, I'm so sorry. I overlooked it when I saw the document PR.

taniguchiiqqq commented on backend/main/src/Shared/Adapters/DB/NishikiDBTypes.ts:
```diff
@@ -43,3 +43,58 @@ export type InvitationLink = {
+
+export type Food = {
+	foodId: string;
+	name: string | null;
```
I think the type of Name should be string because the name is a required prop. Or is there any case it can be null if we use dynamoDB? ( same comment as another one)

taniguchiiqqq reviewed (commented):
Thank you for implementing! LGTM except a comment.

hitohata committed:
fix: food name's type

hitohata committed:
Merge remote-tracking branch 'origin/develop' into feature/150-getcontainer

hitohata committed:
fix: the food id in the UpdateFoodOfContainerUseCase.test.ts

hitohata commented:
I fixed the food data type and fixed its document.
https://github.com/genesis-tech-tribe/nishiki-documents/pull/115

I also found that the food id test failed, so I fixed it.

taniguchiiqqq reviewed (approved):
Thank you! LGTM!
